src/components/designObject.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+    <div class="vertical">
+
+        <input class="min-width-small height-20" :placeholder="el.label" v-model="el.value"></input>
+
+
+</div>
+
+
+</template>
+
+<script>
+export default {
+    name: 'obj',
+    props: ['el'],
+
+    methods: {
+        fireAddCol() {
```

Comment:
We don't need these event dispatchers since we are not adding/removing rows/cols in DESIGN mode.